log/development.log

```diff
@@ -4406,3 +4406,77 @@ Processing by Rails::WelcomeController#index as HTML
 Completed 200 OK in 15ms (Views: 6.8ms | ActiveRecord: 0.0ms)
 
 
```

Comment:
add this directory to gitignore

Reply:
Directory log added to gitignore
![removeLogDirectory](https://user-images.githubusercontent.com/58991676/71357940-06d90800-2566-11ea-9155-d29ba21da30c.png)